### ISSUE TITLE
jspm: add color-coding for "out-of-date but most recent in semver range"

### ISSUE THREAD

**Issue by Thanood** [monterey-framework/monterey#71]
If I have a package defined as `babel-core@^5.8.24` with the newest version being `babel-core@6.10.4` and I have installed `babel-core@^5.8.38` then the grid should indicate that I don't use the latest version but the latest verson available in the version range specified.

This could IMO be done with the `semver` package which is already imported (I believe it's called `semver.satisfies(...)`).

**Comment by Thanood** [comment]
@JeroenVinke I can try to implement this if you like. :smile:

**Comment by JeroenVinke** [comment]
sure! that would be cool

**Comment by JeroenVinke** [comment]
One issue I found though was that npm only shows the latest tag and beta tags which would make this difficult

**Comment by Thanood** [comment]
Yep, that's a problem.. Somehow need to find the "latest in semver range" with npm.
